jest.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  testRegex: './*\\.test\\.tsx$',
+  testRegex: './*\\.test\\.tsx?$',
```

Comment:
For understanding, what is this change improves here?

Reply:
This now gets the `MeetingManager.ts` into testing, earlier it wont include `.ts` files into testing when ran.